hw6/6.1/list.h

```diff
@@ -0,0 +1,19 @@
+#ifndef INC_6_1_LIST_H
+#define INC_6_1_LIST_H
+
+typedef int Value;
+
+typedef struct List {
+    Value value;
+    struct List *next;
+} List;
```

Comment:
```suggestion
typedef struct SortedList SortedList;
```
Внутреннее представление типа должно быть скрыто в .c-файле. А в .h-файле должно быть только объявление типа.
И это ведь не просто список, это отсортированный список.